ctapipe/io/dl1eventsource.py

```diff
@@ -35,7 +35,7 @@
 logger = logging.getLogger(__name__)
 
 
-COMPATIBLE_DL1_VERSIONS = ["v1.0.0", "v1.0.1", "v1.0.2", "v1.0.3", "v1.1.0"]
+COMPATIBLE_DL1_VERSIONS = ["v1.0.0", "v1.0.1", "v1.0.2", "v1.0.3", "v1.1.0", "v1.2.0"]
```

Comment:
We didn't actually change anything in the data model, right?

Reply:
Kind of. Using the stage1-tool with `allowed_tels` now leads to a different file as the subarray is different

Reply:
No, this is not a change to the data model.